aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/BaseHandlerStd.java

```diff
@@ -308,38 +322,17 @@ protected boolean isAssociatedRolesEqual(
         }
 
         return progress;
+    }
+
+    private void logResponse(final LoggerWrapper loggerWrapper,
+                             final ProgressEvent<ResourceModel, CallbackContext> progressEvent) {
+        loggerWrapper.log("Response ProgressEvent: ", progressEvent, SENSITIVE_PARAMETERS_PARENTS);
+        loggerWrapper.log("Response ResourceModel: ", progressEvent.getResourceModel(), SENSITIVE_PARAMETERS);
+    }
 
-        /*
-        return proxy.initiate("rds::tag-dbcluster", proxyClient, progress.getResourceModel(), progress.getCallbackContext())
-                .translateToServiceRequest(Translator::describeDbClustersRequest)
-                .makeServiceCall((describeRequest, rdsClientProxyClient) -> rdsClientProxyClient.injectCredentialsAndInvokeV2(
-                        describeRequest,
-                        rdsClientProxyClient.client()::describeDBClusters
-                ))
-                .handleError((describeRequest, exception, client, model, context) -> Commons.handleException(
-                        ProgressEvent.progress(model, context),
-                        exception,
-                        DEFAULT_DB_CLUSTER_ERROR_RULE_SET
-                ))
-                .done((describeRequest, describeResponse, rdsClientProxyClient, resourceModel, context) -> {
-                    final DBCluster dbCluster = describeResponse.dbClusters().stream().findFirst().get();
-                    final String arn = dbCluster.dbClusterArn();
-
-                    final Set<Tag> currentTags = Optional.ofNullable(resourceModel.getTags()).orElse(Collections.emptySet());
-                    final Set<Tag> existingTags = Translator.translateTagsFromSdk(dbCluster.tagList());
-                    final Set<Tag> tagsToRemove = Sets.difference(existingTags, currentTags);
-                    final Set<Tag> tagsToAdd = Sets.difference(currentTags, existingTags);
-
-                    rdsClientProxyClient.injectCredentialsAndInvokeV2(
-                            removeTagsFromResourceRequest(arn, tagsToRemove),
-                            rdsClientProxyClient.client()::removeTagsFromResource
-                    );
-                    rdsClientProxyClient.injectCredentialsAndInvokeV2(
-                            addTagsToResourceRequest(arn, tagsToAdd),
-                            rdsClientProxyClient.client()::addTagsToResource
-                    );
-                    return ProgressEvent.progress(resourceModel, context);
-                });
-         */
+    private void logRequest(final LoggerWrapper loggerWrapper, final ResourceHandlerRequest<ResourceModel> request) {
+        loggerWrapper.log("Request: ", request, SENSITIVE_PARAMETERS_PARENTS);
+        loggerWrapper.log("DesiredResourceState: ", request.getDesiredResourceState(), SENSITIVE_PARAMETERS);
+        loggerWrapper.log("PreviousResourceState: ", request.getPreviousResourceState(), SENSITIVE_PARAMETERS);
```

Comment:
I believe we could rather inline these methods, as the approach pushes one to add more custom object-specific logging functions, which could pollute the code.